src/plugins/dev_tools/public/application.tsx

```diff
@@ -42,6 +44,11 @@ function DevToolsWrapper({ devTools, activeDevTool, updateRoute }: DevToolsWrapp
     []
   );
 
+  useEffect(() => {
+    docTitleService.setTitle(activeDevTool.id as BreadcrumbType);
+    breadcrumbService.setBreadcrumbs(activeDevTool.id as BreadcrumbType);
```

Comment:
These plugin id's are all either lowercase or snakecased, so I ended up using the exact same keys in the types and in the breadcrumbs hashtable to avoid having to go and rename them in all the other plugins + fix the broken tests that might cause.